docs/development/code_review.md

```diff
@@ -0,0 +1,81 @@
+# Code review
+
+This document describes
+general guidelines for developers
+when reviewing a Pull Request (PR).
```

Comment:
Afterwards you switch between writing `Pull Request` or `PR`. Use one or the other consistently.